### DISCUSSION TITLE
SO 2.3.210 - installer-screen goes blank shortly after "Starting installer..."-message (Installation on Hardware)

### DISCUSSION THREAD

**Discussion by ejgh-oe** [Security-Onion-Solutions/securityonion#9755] [Unsupported Versions] [answered]
Hi,
Today I wanted to install SO 2.3.210 on a relatively new hardware (HP Proliant Gen10 Plus, latest BIOS, 128GB RAM, several SSDs). In an early phase of the installation, seconds after the message "Starting installer, one moment..." appears on top of the screen and "[anaconda] 1:main..." on the bottom the screen goes blank - and then - nothing :-(. The screen stays blank - even after almost an hour - nothing.

Things I already checked/tried:

- ISO: Is OK as per the sha256 checksum
- Tried installing both via the HP "iLO" (remote console) and directly at the server using Monitor, Kbd and mouse
- UEFI vs. BIOS
- adding "nomodeset text" to the boot commandline

None of that changed anything - same symptoms as described above.

BTW, the server hardware is OK; installing any other OS (tried Ubuntu, CentOS, FreeBSD) went OK.

Any ideas as to what can be wrong here?

Thanks much in advance for any clue...

**Comment by ejgh-oe** [comment, marked as answer]
Hi,

This is a followup to me question about wrt. boot problems of SecurityOnion 2.3.210 on a HP Gen10+ Hardware Server.

TL;DR;
A colleague of mine was able to track down the root cause of the problem being the SO-installer ISO unable to handle the RAID-controller (Broadcom MegaRAID MR416i-a) that comes with the server. As a workaround he was able to build a "homebrew" SO-installer-ISO that would boot on the server.

Details:
While booting from the original installer-ISO (securityonion-2.3.210-20230202.iso) an error message "line 65: /3: syntax error: operand expected (error token is "/3")" is briefly displayed on the console. This message is, aside from the line number, almost identical to the one mentioned in https://github.com/Security-Onion-Solutions/securityonion/issues/1825

Working theory was, that the version of CentOS that comes with SecurityOnion is too old to detect the relatively new Broadcom MegaRAID controller on our server.

To give it a shot my colleague went ahead by modifying the OS Installer-ISO: replacing initrd.img, vmlinuz, LiveOS/squashfs.img on from the SO-installer ISO with more recent versions of the same files from a more recent version of a plain vanilla CentOS7-install-ISO.

Using this modified ISO we were able to boot the server and install SO :-)

So to summarize it really looks like the SO installer is based on an older version of CentOS 7 that cannot handle newer hardware components.

Maybe with the next release of 2.3.xxx the SO-team could provide an updated installer that can handle newer hardware out of the box?
